nexus/src/external_api/http_entrypoints.rs

```diff
@@ -233,11 +233,11 @@ pub fn external_api() -> NexusApiDescription {
         api.register(silo_identity_provider_create)?;
         api.register(silo_identity_provider_view)?;
 
-        api.register(image_global_list)?;
-        api.register(image_global_create)?;
-        api.register(image_global_view)?;
-        api.register(image_global_view_by_id)?;
-        api.register(image_global_delete)?;
+        api.register(system_image_list)?;
+        api.register(system_image_create)?;
+        api.register(system_image_view)?;
+        api.register(system_image_view_by_id)?;
+        api.register(system_image_delete)?;
```

Comment:
We need some way to differentiate this, but I do wonder if this prefix is correct given that we're not using prefixes for other things.

Reply:
Punt. I think it's tolerable.